### PR TITLE
 pickled favorites

### PR DESCRIPTION
**Pull request by dandre2**

### CODE REVIEW

File: DA/submissions/pickled favorites [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import pickle
+favorites = ['football', 'wrestling', 'weightlifting', 'coding']
+f = open('favorites.dat', 'wb')
+pickle.dump(favorites, f)
+f.close()
+load_favourites= open('favorites.dat', 'rb')
```
Comment (rakirs2) [inline DA/submissions/pickled favorites:6]: why are there two spellings of favorites lol. I'm all for the british english spelling but lets stick to one :)

File: DA/submissions/pickled favorites [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import pickle
+favorites = ['football', 'wrestling', 'weightlifting', 'coding']
```
Comment (rakirs2) [inline DA/submissions/pickled favorites:2]: So let's add a comment here telling us what the problem is and what we hope to achieve in this code

**Review by rakirs2 — CHANGES_REQUESTED** [review]
If possible, for the book problems, just give me the problem statement as a comment @ the top

File: DA/submissions/pickled favorites (hunk 1 above)
Comment (dandre2) [reply DA/submissions/pickled favorites:6]: fixed

File: DA/submissions/pickled favorites (hunk 2 above)
Comment (dandre2) [reply DA/submissions/pickled favorites:2]: done

**Pull request closed without merging**